<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stats</class>
 <widget class="QFrame" name="Stats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Gems</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Base Health</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gem_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hp_label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
